Mathlib/Data/Set/Image.lean

```diff
@@ -492,6 +492,17 @@ theorem image_preimage_subset (f : α → β) (s : Set β) : f '' (f ⁻¹' s) 
   image_subset_iff.2 Subset.rfl
 #align set.image_preimage_subset Set.image_preimage_subset
 
+-- Phrased in terms of f '' univ, as `Set.range` is only defined further below.
+theorem image_preimage_eq_subset' {f : α → β} {s : Set β} (hs : s ⊆ f '' univ) :
```

Comment:
What's wrong with using [Set.image_preimage_eq_of_subset](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Image.html#Set.image_preimage_eq_of_subset)?

Reply:
I didn't find it :-) Thanks for pointing it out!